### ISSUE TITLE
allow adding t-ref on text nodes

### ISSUE THREAD

**Issue by ged-odoo** [odoo/owl#1470]
something like this:
```xml
<t t-ref="someRef">text</t>
```

**Comment by sdegueldre** [comment]
Not necessarily against it but curious about the use case. Also this may not be super straight forward as this probably also means we want to support it on `<t t-esc="stuff"/>` nodes which is not the same type of BDom node, and it can generate potentially difficult to understand errors when the node has more than one child node, including comment nodes:
```xml
<t t-ref="brokenRef">
  some text
  <!-- comment explaining something -->
  more text
 </t>
 ```

**Comment by ged-odoo** [comment]
i don't think we should do it now, unless the guy that reports the issue shows a compelling usecase
